examples/example_config_files/public_TrueCoaxial.yaml

```diff
@@ -55,6 +55,8 @@ detectors:
       parameters:
         τh: 1ms
         τe: 1ms
+      model_inactive: ConstantLifetime
+      parameters_inactive:
         τh_inactive: 1μs
         τe_inactive: 1μs
```

Comment:
Again here, I would just call it `τe` and `τh` without the `_inactive` (as this is already in the `parameters_inactive` field name.